### ISSUE TITLE
Typescript error

### ISSUE THREAD

**Issue by cardimajs** [jodit/jodit-react#86]
Hi, my code is:

```
<JoditEditor
  config={{readonly: false}}
/>
```
i i get this error: Type '{ readonly: false; }' is missing the following properties from type 'Config': iframe, commandToHotkeys, license, preset, and 56 more.ts(2740)

![image](https://user-images.githubusercontent.com/5782144/91145853-7884cb00-e68c-11ea-8d2d-694a149df95e.png)

how can i fix?

thank you.

**Comment by adamayd** [comment]
I'm sure this isn't the best solution, but I left out `config` and `tabIndex` from the component attributes and it renders.  My project runs Typescript so it also error until I added `experimentalDecorators: true` in my `tsconfig.json` file.

**Comment by cardimajs** [comment]
@adamayd hi,
this is not a solution for me, i need set some config.
looking at source code i can find file index.d.ts, this file import typescript interface IJodit from Jodit in line 2
`import { IJodit } from "jodit";`
looks the problem is not in this repo but in the main Jodit repo types definition, anyone can confirm that?

**Comment by cyapa** [comment]
> I'm sure this isn't the best solution, but I left out `config` and `tabIndex` from the component attributes and it renders. My project runs Typescript so it also error until I added `experimentalDecorators: true` in my `tsconfig.json` file.

Yes I had to update the tsconfig.json as well but I don't want to specify all 56+ configs :/

**Comment by Crocsx** [comment]
How is this still an Issue 5 months later ? I just added this package to my project cause IJodot is really nice, but this wrapper as so many type missconfiguration.

**Comment by AnimalInstinct** [comment]
Ok, i see just installed this editor and as i can see old bug still not fixed, piece of crap... next...

**Comment by Pfahlf** [comment]
Same here, fresh installation and problems...

**Comment by Fedeorlandau** [comment]
Same here, the config object requires a lot of information that is not necessary

**Comment by niksonleonard** [comment]
Hi every one!

I get it working, just get default config and spread it with custom configs:

`import { IJodit, Jodit } from "jodit"`

`
const config: IJodit['options'] = {
    ...Jodit.defaultOptions,
    theme: 'dark'
}
`

**Comment by samuelfsilva** [comment]
> Hi every one!
> 
> I get it working, just get default config and spread it with custom configs:
> 
> `import { IJodit, Jodit } from "jodit"`
> 
> `const config: IJodit['options'] = { ...Jodit.defaultOptions, theme: 'dark' }`

It works for me!

**Comment by Jeff909Dev** [comment]
> Hi every one!
> 
> I get it working, just get default config and spread it with custom configs:
> 
> `import { IJodit, Jodit } from "jodit"`
> 
> `const config: IJodit['options'] = { ...Jodit.defaultOptions, theme: 'dark' }`

it works!

**Comment by Mulperi** [comment]
If you want to use "toolbarAdaptive" configuration attribute, you will get **TypeScript error: 'toolbarAdaptive' does not exist in type 'Config'**
https://github.com/jodit/jodit-react/issues/32

**Comment by MelvinCado** [comment]
Hello,

I had got the same error.
So to add some config, i used the 'any' type.

```    
const config: any = {
  readonly: false,
  placeholder: 'Start typings...',
  height: 500,
  style: { backgroundColor: 'red' },
}
```

To put the config on the component without errors.

```
<JoditEditor
    ref={editor}
    config={config}
    value={description}
    onBlur={(newContent) => setDescription(newContent)}
    onChange={(newContent) => setDescription(newContent)}
/>
```

Like this you can use the component normally during waiting them fixes.
